docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,59 @@
+
+## The difference between core and example modules
+
+**Core modules** provide generic, reusable functionality that benefits a wide range of customers. They offer a foundation for customization and serve as versatile building blocks.
+
+**Example Modules** in contrast, are designed for specific, one-off use-cases. Their functionality is tightly aligned with unique business requirements and isn't suitable for broad reuse. These modules serve as specialized, project-specific solutions.
+
+## Example module use-cases
+
+### SCOS: Showcase functionality
+
+SCOS uses example modules to fill the gap with concrete/example implementation over an abstract workflow (latter is the target of delivery), where the concrete/example implementation is never actually usable by any concrete customer or business use-case, thus the actual business logic is not part of the core (eg: the React API Example).
```

Comment:
do you mean that the React API example is the example of an app with the example modules? Or anything else?

Reply:
Yes, but I just realized it was removed. I'll update this and use another module here.